spec/javascripts/components/story/expanded_story/expanded_story_spec.js

```diff
@@ -1,116 +1,141 @@
-import React from 'react';
-import { shallow } from 'enzyme';
+import { screen } from '@testing-library/react';
 import { ExpandedStory } from 'components/story/ExpandedStory/index';
+import React from 'react';
 import storyFactory from '../../../support/factories/storyFactory';
-import ExpandedStoryRelease from 'components/story/ExpandedStory/ExpandedStoryRelease';
-import ExpandedStoryDefault from 'components/story/ExpandedStory/ExpandedStoryDefault';
+import { renderWithProviders } from '../../setupRedux';
 
 describe('<ExpandedStory />', () => {
-  const defaultProps = () => ({
-    story: storyFactory(),
-    editStory: vi.fn(),
-    saveStory: vi.fn(),
-    deleteStory: vi.fn(),
-    project: { labels: [] },
-    disabled: false,
-    onToggle: vi.fn(),
-    cloneStory: vi.fn(),
-    showHistory: vi.fn(),
-    onClone: vi.fn(),
-  });
+  vi.spyOn(window.md, 'makeHtml').mockReturnValue('<p>Test</p>');
+  vi.mock('react-clipboard.js', () => ({
+    default: vi
+      .fn()
+      .mockImplementation(({ children }) => <div>{children}</div>),
+  }));
+
+  const renderComponent = props => {
+    const defaultProps = {
+      story: storyFactory(),
+      editStory: vi.fn(),
+      saveStory: vi.fn(),
+      deleteStory: vi.fn(),
+      project: { labels: [], pointValues: [] },
+      disabled: false,
+      onToggle: vi.fn(),
+      cloneStory: vi.fn(),
+      showHistory: vi.fn(),
+      onLabelClick: vi.fn(),
+      onClone: vi.fn(),
+    };
+
+    const mergedProps = { ...defaultProps, ...props };
+
+    return renderWithProviders(<ExpandedStory {...mergedProps} />);
+  };
 
   describe('when storyType is a release', () => {
     it('renders ExpandedStoryRelease component', () => {
-      const story = storyFactory({
-        _editing: {
-          ...storyFactory(),
-          storyType: 'release',
-        },
-      });
-
-      const wrapper = shallow(
-        <ExpandedStory {...defaultProps()} story={story} />,
-        { disableLifecycleMethods: true }
+      const props = {
+        story: storyFactory({
+          _editing: {
+            ...storyFactory(),
+            storyType: 'release',
+          },
+        }),
+      };
+
+      renderComponent(props);
+      const releaseTitle = screen.getByText(
+        I18n.t('activerecord.attributes.story.release_date')
       );
+      console.log(releaseTitle.innerHTML);
```

Comment:
Can we remove this line?

Reply:
Done!

[81eeaff](https://github.com/Codeminer42/cm42-central/pull/926/commits/81eeaffce9b0187e64e8973a07e71f152b5e370f)